### PR TITLE
Misc DB fixes 2

### PR DESCRIPTION
**Pull request by kampeador**
Misc DB fixes 2:
- Fixed quest The Fargodeep Mine(id:62), area triggers 88 and 197. https://github.com/elysium-project/server/issues/1517
- Fixed quest The Jasperlode Mine(id:76), area triggers 87 and 342.
- Fixed quest Verifying the Corruption(id:5156), area triggers 2206, 2207, 2208.
- Before patch 2.4, quest Milly Osworth(id:3903) was available after Brotherhood of Thieves(id:18).
- Removed outdated lockpicking trainers. Lockpicking cap is increased with player's level. https://github.com/elysium-project/server/issues/1472
- Fixed Tabetha's and Merissa Stilwell's models.
- Lieutenant Doren had a mount before TBC.
- Removed a winterspring owl from Silithus.
- Fixed a loot table for Small Barnacled Clam. https://github.com/elysium-project/server/issues/1461
- Fixed an Archmage Malin gossip, removed a script attached to him. That quest script is from TBC.
- Fixed a mount model for Executor Zygand.
- Fixed a mount model for Deathguard Lundmark.
- Rabbits can also be brown.
- Rats can also be brown and white.
- Sheep can also be grey.
- Fixed an equipment flag for Brother Benjamin.
- Added waypoints for Mabel Solaj from Darkshire Inn.
 - Felwood Slime Sample(item_id:12230) and Un'Goro Slime Sample(item_id:12235) should drop only during quest and their chance should be 100%. https://github.com/elysium-project/server/issues/1559
- Fixed Felwood Slime Sample(id:12230) drop rate for The Ongar(npc_id:14345)
- Ogres shouldn't drop Pure Un'Goro Sample(id:12236). It shouldn't drop from mobs at all.
- Added gossip texts for "Testing equipment" objects.
- Fixed faction id for The Ongar(npc_id:14345).
- Removed duplicate "Testing Equipment" gameobject spawns.
- Fixed skinning loot for Bloodshot(id:11614). https://github.com/elysium-project/server/issues/1565
- All graveyards in the Hinterlands should be cross-faction.
- Fixed graveyard at Bloodhoof Village in Mulgore for the Alliance.
- Fixed George Candarte's(id:2698) model. https://github.com/elysium-project/server/issues/1577
- Enable graveyard at Darkshore, Twilight Vale. It should be cross-faction.
- Enable Undercity graveyard.
- Fixed quest level (id:3181 The Horn of the Beast). It was 48 lvl before wotlk. https://github.com/elysium-project/server/issues/1678
- Fixed typo in text for quest (id:3092 Etched Note). https://github.com/elysium-project/server/issues/1615
- Fixed texts for some kobolds. You no take candle!
- Removed random Defias Brotherhood texts from the mobs that shouldn't have it. https://github.com/elysium-project/server/issues/1830

### CODE REVIEW

**Comment by thepparker** [comment]
Proof of Felwood Slime Sample drop rate https://youtu.be/AnAjOxoMD9E?t=315

**Comment by brotalnia** [comment]
@kampeador I have gone through your changes and i have a few questions.

- Merissa Stilwell
What is the source for the model change? Screenshots from WoWHead and WoWWiki show her with the current model.
http://wowwiki.wikia.com/wiki/Merissa_Stilwell?oldid=873471
http://wow.zamimg.com/uploads/screenshots/normal/73660-merissa-stilwell.jpg
Also the model you assigned her 5375 is used by another NPC, Tabetha in Dustwallow Marsh. Both displayids exist in the 1.0.0 client (freshly installed from old disk) so it's not like the current one was added later. She has the same model in both cMaNGOS-Classic and mangosZero databases.

- Lieutenant Doren
Again, i can't find any sources indicating that he had a mount. You can see that he doesn't have one in these screenshots from 2006.
http://wowwiki.wikia.com/wiki/Lieutenant_Doren?oldid=317495
http://wowwiki.wikia.com/wiki/Rebel_Camp?oldid=319109

- Archmage Malin
When you unassign a script in the db, you should remove it from the core too because leaving it produces a startup error about an unused script.
https://github.com/elysium-project/server/blob/af4eb1793829d1666921df5d064a11fb427fbe47/src/scripts/eastern_kingdoms/stormwind_city/stormwind_city.cpp#L35

**Comment by kampeador** [comment]
I haven't updade this for a while.

1. If you look in `creature_model_info` table, you will see that:
5375 has modelid_other_gender 11898
11898 has modelid_other_gender 5375
Sometimes, gender detection goes wrong. I am pretty sure if you login on live servers now you will see her "wrong" model. 
This fix just make no sence. I'll revert this.

2. Unfortunately, you won't find any sources about this, i took it from 2 databases which are older than those screenshots. There were alot of hidden changes around in the game world. During 1.x WoW emulation was not as public as it is today. There were even some closed sourced commercial emulators which you could license.
It is pre 1.10(28 March 2006), but i can't give you exact date, exact patch.

3. I didn't want to mix db fixes with c++ code. There are some other TBC scripts enabled. I think its better disable them with the separate PR.

**Comment by martinjlowm** [comment]
Regarding Lieutenant Doren, [1] also shows him without a mount. Now, I do know this is after the said date, but I believe it's worth to have around as a source.

[1] https://www.youtube.com/watch?v=CvSHarP-Zjc

File: sql/migrations/20170717113000_world.sql [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+DELETE FROM `creature_ai_scripts` WHERE `comment` = 'Bloodsail Warlock - Random Aggro Say';
+DELETE FROM `creature_ai_scripts` WHERE `comment` = 'Burning Blade Apprentice - Random Aggro Say';
+
+-- Fix texts for some kobolds. You no take candle!
+-- Gogger Rock Keeper - Random Say on Aggro
+UPDATE `creature_ai_scripts` SET `action1_param1` = -7, `action1_param2` = 0, `action1_param3` = 0 WHERE `id` = 1191501;
+-- Gogger Geomancer - Random Say on Aggro
+UPDATE `creature_ai_scripts` SET `action1_param1` = -7, `action1_param2` = 0, `action1_param3` = 0 WHERE `comment` = 1191702;
+-- Gogger Stonepounder - Random Say on Aggro
+UPDATE `creature_ai_scripts` SET `action1_param1` = -7, `action1_param2` = 0, `action1_param3` = 0 WHERE `comment` = 1191801;
```
Comment (brotalnia) [inline sql/migrations/20170717113000_world.sql]: This does nothing. You wrote `comment` instead of `id` for the column name.

**Comment by brotalnia** [comment]
And about Lieutenant Doren's mount, if it did not exist before 1.10, then report it in the database repository. It should be added to the progression patches, not as a migration. Remove the mount change from this pull request and I can merge it.

https://github.com/elysium-project/database

**Pull request merged**